x-pack/plugins/security/server/authentication/authentication_service.test.ts

```diff
@@ -128,6 +128,17 @@ describe('AuthenticationService', () => {
         expect.any(Function)
       );
     });
+
+    it('properly registers auth handler with no providers', () => {
+      const mockAuthGet = mockStartAuthenticationParams.http.auth.get as jest.Mock;
+      mockAuthGet.mockReturnValue({ sortedProviders: [] });
+      service.setup(mockSetupAuthenticationParams);
```

Comment:
__question:__ what do we try to test with this test? I see this test passes even without the fix, I guess that's because `mockSetupAuthenticationParams.config` is by default configured with the `basic` provider. 

Would you mind overriding `mockSetupAuthenticationParams.config` in this test to not have any providers so that test fails without the fix and passes with it?

Reply:
yes!